### PR TITLE
pkg(samsung): add com.samsung.android.adaptivebrightnessgo

### PR DESCRIPTION
**Pull request by jxdv**

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -1800,6 +1800,14 @@
     "labels": [],
     "removal": "Recommended"
   },
+  "com.samsung.android.adaptivebrightnessgo": {
+    "list": "Carrier",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:1804]:
```suggestion
    "list": "Oem",
```

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -1800,6 +1800,14 @@
...
+    "description": "Samsung Adaptive Brightness\nFeature of Samsung devices that, as the name suggests, automatically adjusts the brightness level of display based on environmental lighting.\nThis feature is available even after uninstalling.",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:1805]: Does it also work correctly instead of the feature still being there?

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: resources/assets/uad_lists.json (hunk 2 above)
Comment (jxdv) [reply resources/assets/uad_lists.json:1805]: Yes, I tested this by uninstalling this package and turning on the adaptive brightness feat. The display brightness still gets adjusted normally.

**Review by AnonymousWP — APPROVED** [review]
Thanks again. 🙂

**Pull request merged**